manifests/server/config.pp

```diff
@@ -142,8 +142,11 @@
 
     ensure_packages([$package_name])
 
+    $exec_command = ['/usr/sbin/semanage', 'port', '-a', '-t', 'postgresql_port_t', '-p', 'tcp', $port]
+    $exec_unless = ["/usr/sbin/semanage port -l | grep -qw ${port}"]
```

Comment:
Note this will still call a shell. In this case it's safe, but good to remember.

Reply:
👍

Reply:
It does make me wonder: should this actually just be a string and kept as-is? That does make it more explicit what it is and the diff smaller.

Reply:
thats a fair point, swapping it to a string